tests/data_files/fixed/crl-ec-sha1.pem

```diff
@@ -0,0 +1,10 @@
+-----BEGIN X509 CRL-----
```

Comment:
The commit descriptions say “move files …”, but the commits are only adding files. Are the commits missing `git rm` of the old files?

Reply:
I have re-written the history and this comment might be out of data